SampleApplications/2023/ServerlessDocumentAnalysis/functions/test/DocProcessingTest/TestTextractQuery.cs

```diff
@@ -0,0 +1,64 @@
+using DocProcessing.Shared.Model.Textract.QueryAnalysis;
+using System.Text.Json;
+
+
+namespace DocProcessingTest;
+
+[TestClass]
+[DeploymentItem(@"TestAssets\TextractResults.json")]
+public class TestTextractQuery
+{
+    private TextractDataModel? TextractData { get; set; }
+    private TextractAnalysisResult? TextractResult { get; set; }
+
+    [TestInitialize()]
+    public void Setup()
+    {
+        using FileStream jsonStream = File.OpenRead(@"TextractResults.json");
+        if (jsonStream is null)
```

Comment:
Should neve have condition in tests, preferably use Asserts

Reply:
Good call. Updating.